### ISSUE TITLE
Proposal: add SocialMedia to contactPointOption

### ISSUE THREAD

**Issue by WeaverStever** [schemaorg/schemaorg#3596]
It is generally accepted that social media urls are not appropriate in the sameAs property.   

As mentioned in #3403, I came to the realization that social media accounts are actually contactOptions, and in many cases, the only (or preferred) contact options.  (The op in #3403 is looking for a creation date property for social media profiles).

Currently, [ContactPointOption](https://schema.org/ContactPointOption) has two enumerations for use in contactOptions.

Enumeration members in ContactPointOption

- [HearingImpairedSupported](https://schema.org/HearingImpairedSupported)
- [TollFree](https://schema.org/TollFree)

I suggest adding the third enumeration

- SocialMedia

(In the example below, we could use a startTime property into potentialAction to indicate the creation date of the social media acct.)

~~~
<script type="application/ld+json">
{ "@context" : "https://schema.org",
  "@type" : "Organization",
  "url" : "http://www.t-mobile.com",
  "contactPoint" : [
    { "@type" : "ContactPoint",
      "telephone" : "+1-877-746-0909",
      "contactType" : "customer service",
      "contactOption" : "TollFree",
      "areaServed" : "US"
    } , {
      "@type" : "ContactPoint",
      "telephone" : "+1-505-998-3793",
      "contactType" : "customer service"
    } , {
      "@type" : "ContactPoint",
      "alternateName" : "@TMobile",
      "contactType" : "Twitter account",
      "contactOption" : ["SocialMedia"],
      "url": "https://x.com/tmobile",
      "potentialAction": {
        "startTime": "1999-12-01"     
      }
    }
  ] 
}
</script>
~~~

**Comment by MatthiasWiesmann** [comment]
Would it make sense to make `ProfilePage` inherit from ContactPoint.

**Comment by WeaverStever** [comment]
@MatthiasWiesmann 

it appears that Google wants Persons and / or Organizations beneath the MainEntity @type.  I would seem to me that contactPoint would always be associated with (beneath) a person or organization. A profilePage could be for virtually anything. Am I wrong???

https://developers.google.com/search/docs/appearance/structured-data/profile-page

**Comment by MatthiasWiesmann** [comment]
I would assume that when we talk about a profile page, we are talking about the page on a social network (or something similar) that describes an entity that interacts on that network, these would be either a private person, or a company. I don't think ProfilePage would be appropriate for say a Book.

**Comment by WeaverStever** [comment]
@MatthiasWiesmann 

From [Google](https://developers.google.com/search/docs/appearance/structured-data/profile-page)

> ProfilePage markup is designed for any site where creators (either people or organizations) share first-hand perspectives. Adding this markup helps Google Search understand the creators that post in an online community, and show better content from that community in search results, including the [Discussions and Forums](https://blog.google/products/search/google-search-discussions-forums-news/) feature.

[Yoast says](https://developer.yoast.com/features/schema/pieces/webpage/#profile-pages)

> `Profile pages
When the page is about a particular user (e.g., an member bio / author archive): 
 The type property should be altered to ProfilePage. 
If the page features posts/pages authored by that Person, then the type property should instead by an array of ['CollectionPage','ProfilePage'].
Add a mainEntityOfPage property to the Person which the page is about, which references the WebPage by ID.

The new version of schema.org just says "Web page type: Profile page."

Interesting that Yoast points out that a profile page may be for a user on a social media site. From the perspective of a personal website, the profile page would make sense for linking to social media profiles. 

In the case I am proposing here, my clients have social media links in the sidebar or footer on every page of the website,  so I'm looking for an official way to describe that these are social media contactPoints for the person / organization -- not just for use on the profile page. (Which may or may not get visits).

**Comment by WeaverStever** [comment]
Giving this a bump.

Most of my website clients use social media as their main contact points.  It would be nice to be able to explain to search engines what those icons in the footer are.  For musicians, the icons are intermixed with icons to music streaming sites.

Enumerating "Social Media"  in   [ContactPointOption](https://schema.org/ContactPointOption) would do the trick.

Finally, my understanding is that social media urls are unwanted as SameAs items because they don't really describe the person / organization.

**Comment by WeaverStever** [comment]
I gave this a bump last week, but I think it got buried in our spam episode.

Any more ideas on how to describe social media as contact points? Other?

**Comment by WeaverStever** [comment]
Since the socialMedia enumeration for ContactPointOption discussion did not get any traction.  I decided to kluge the socials into a BroadcastService definition.

Example: 

~~~~
<script type="application/ld+json"> 
  {"@context":"https://schema.org",
  "@type": ["BroadcastService"],
  "serviceType": "Streaming music catalog",  
  "provider": {"@id": "/#musicgroup"},
  "parentservice":[{
    "@type":"BroadcastService",
    "@id": "/#Facebook",
    "name":"Facebook",
    "broadcastDisplayName":"Valerie Jeanne Music on Facebook",
    "url": "https://www.facebook.com/ValerieJeanneMusic/"
    },{
    "@type":"BroadcastService",
    "@id": "/#Instagram",
    "name":"Instagram",
    "broadcastDisplayName":"Valerie Jeanne Music on Instagram",
    "url": "https://www.instagram.com/valeriejeannesong/"
    },{
    "@type":"BroadcastService",
    "@id": "/#YouTube",
    "name":"YouTube",
    "broadcastDisplayName":"Valerie Jeanne Music on YouTube",
    "url": "https://www.youtube.com/channel/UCsyeQEqvRRBGKLCLmGjMvqQ"}]
  }
</script>
~~~~

**Comment by jgoodell2** [comment]
Yes, it seems to me that ContactPoint should include a way of specifying one or more social media account handles, e.g. 

  "contactPoint" : [
    {
      "@type" : "ContactPoint",
      "contactType" : "LinkedIn",
      "contactOption" : ["SocialMedia"],
      "url": "https://www.linkedin.com/in/jimgoodell/"
    }
  ]

**Comment by jgoodell2** [comment]
Or maybe the solution is to use the exiting "contactType" but have a an example showing a best practice or consistent convention or add controlled vocabulary for specifying the social media platform in contactType. e.g. "LinkedIn" vs. "linked in" vs. "linked-in".

"contactPoint" : [
{
"@type" : "ContactPoint",
"alternateName": "@jimgoodell",
"contactType" : "LinkedIn",
"contactOption" : ["SocialMedia"],
"url": "https://www.linkedin.com/in/jimgoodell/"
}
